src/app/account.service.ts

```diff
@@ -460,11 +544,12 @@ export class AccountService {
       network,
       loginMethod,
       version: PrivateUserVersion.V2,
+      lastLoginTimestamp: Date.now(),
```

Comment:
This is only for the case where you use one of the signup options (rather than the account selector). If you use the account selector, we updated it directly. This is where that happens: https://github.com/deso-protocol/identity/blob/1c88ce3485461250fc41c9de202ce7e4e26375bc/src/app/grouped-account-select/grouped-account-select.component.ts#L173